File: src/main/java/me/nobaboy/nobaaddons/features/chatcommands/ChatContext.java
package me.nobaboy.nobaaddons.features.chatcommands;

public class ChatContext {
    private final String user, command, fullMessage;
    private final String[] args;

    public ChatContext(String user, String command, String[] args, String fullMessage) {
        this.user = user;
        this.command = command;
        this.args = args;
        this.fullMessage = fullMessage;
    }

    public String user() {
        return this.user;
    }

    public String command() {
        return command;
    }

    public String[] args() {
        return args;
    }

    public String fullMessage() {
        return fullMessage;
    }
}

File: src/main/kotlin/me/nobaboy/nobaaddons/features/chatcommands/ChatContext.kt
package me.nobaboy.nobaaddons.features.chatcommands

class ChatContext(
    private val user: String,
    private val command: String,
    private val args: List<String>,
    private val fullMessage: String
) {
    fun user(): String {
        return user
    }

    fun command(): String {
        return command
    }

    fun args(): List<String> {
        return args
    }

    fun fullMessage(): String {
        return fullMessage
    }
}